src/webgpu/api/validation/encoding/cmds/index_access.spec.ts

```diff
@@ -129,19 +129,23 @@ class F extends ValidationTest {
 export const g = makeTestGroup(F);
 
 g.test('out_of_bounds')
-  .params(
+  .cases(
```

Comment:
hm, probably should split this into cases and subcases while we're here. I figure only 'indirect' should be a case, the rest can be subcases because they're not very distinct/separable.